### DISCUSSION TITLE
[xcb] error message and rustdesk crash only on one machine of two with almost identical software

### DISCUSSION THREAD

**Discussion by aki42** [rustdesk/rustdesk#5862] [Q&A] [answered]
I run a notebook and a desktop computer with openSUSE 15.4. In terms of Mesa and X11 they both have installed - to my knowledge - the same software packages. But while I can connect without any trouble from the desktop to the notebook using rustdesk, trying to open the connection from the notebook to the desktop machine brings up the following error messages:

`[xcb] Unknown sequence number while processing queue
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
rustdesk: xcb_io.c:272: poll_for_event: Zusicherung »!xcb_xlib_threads_sequence_lost« nicht erfüllt.
Abgebrochen (Speicherabzug geschrieben)`

Afterwards rustdesk crashes on the notebook. The connection notebook → desktop is the one that I use(d) more often and it worked in the past (unfortunately I can't tell when it stopped working) …

Any hints, where I might have to have a look at to make to notebook's rustdesk work, too and again?

**Comment by rustdesk** [comment]
full crash log?

**Comment by aki42** [comment]
I don't know where to find the "fullest" log, so I attach the console output and `.local/share/logs/RustDesk/rustdesk_rCURRENT.log`.
[console.log](https://github.com/rustdesk/rustdesk/files/12776905/console.log)
[rustdesk_rCURRENT.log](https://github.com/rustdesk/rustdesk/files/12776909/rustdesk_rCURRENT.log)

**Comment by rustdesk** [comment]
> Gtk-Message: 11:52:00.472: Failed to load module "canberra-gtk-module"
> 
> (rustdesk:25166): Gtk-WARNING **: 11:52:01.050: Theme parsing error: gtk.css:3233:25: 'font-feature-settings' is not a valid property name
> Gtk-Message: 11:52:01.110: Failed to load module "colorreload-gtk-module"
> Gtk-Message: 11:52:01.114: Failed to load module "window-decorations-gtk-module"
> 
> ** (rustdesk:25166): CRITICAL **: 11:52:02.624: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found
> flutter: launch args: []
> flutter: initializing FFI main
> flutter: _appType:main,info1-id:00070555454844c1801add10d86bb655,info2-name:openSUSE Leap,dir:/home/andi/Dokumente
> flutter: _globalFFI init
> flutter: _globalFFI init end
> flutter: registerEventHandler native_ui native_ui
> flutter: pullAb, force:true, quiet:false
> flutter: registerEventHandler callback_query_onlines recent peer
> flutter: registerEventHandler load_recent_peers recent peer
> flutter: pullAb, force:true, quiet:false
> flutter: restore lpos: 800.0/600.0, offset:12.0/141.0
> flutter: handled by uni links: false
> flutter: Bring window 'null' on top
> flutter: [MultiWindowHandler] active window changed: {0}
> [xcb] Unknown sequence number while processing queue
> [xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
> [xcb] Aborting, sorry about that.
> rustdesk: xcb_io.c:272: poll_for_event: Zusicherung »!xcb_xlib_threads_sequence_lost« nicht erfüllt.
>

**Reply by aki42** [reply]
Yes, I know the contents of the file. If your comment is meant to be about `** (rustdesk:25166): CRITICAL **: 11:52:02.624: Failed to read XDG desktop portal settings` - this error appears on the machine where the connection works, too. So it seems to be not as critical as it claims. It's the [xcb] lines that are unique to the machine where the connection fails.
To be clear: I don't think that this is necessarily a bug. Therefore I asked in "General discussions" for ideas. But I get the mood and won't ask again.

**Reply by rustdesk** [reply]
This is for our convenience, no special meaning for you.

**Reply by aki42** [reply]
Ok, thanks a lot for the clarification. Just posting the log content without any context read like "look here, moron, why do you ask, it's all right before your eyes" to me. I'm really glad that I completely misunderstood. ;-)

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/6477#issuecomment-1820569873
